BraveWallet/Crypto/Search/AssetSearchView.swift

```diff
@@ -17,7 +17,7 @@ struct AssetSearchView: View {
   
   var body: some View {
     NavigationView {
-      TokenList(tokens: allTokens) { token in
+      TokenList(tokens: allTokens.filter({ !$0.isErc721 })) { token in
```

Comment:
Lets flip this so that its `$0.isErc20` (that way we can add support explicitly for other token types) rather than imply support when its `isErc721` is false